packages/ra-core/src/storybook/FakeBrowser.tsx

```diff
@@ -0,0 +1,217 @@
+import * as React from 'react';
+import { ReactNode, useEffect, useState } from 'react';
+import {
+    createMemoryHistory,
+    History,
+    Location,
+    createPath,
+    parsePath,
+} from 'history';
+
+export const FakeBrowserDecorator = (Story, context) => {
```

Comment:
could you exlain how to use it in jsDoc?

Reply:
Done